src/main/java/bowling/domain/Score.java

```diff
@@ -20,6 +20,10 @@ public static Score ofStrike() {
         return new Score(STRIKE_OR_SPARE, 2);
     }
 
+    public boolean isCalculateScore() {
+        return remainingNumber == 0;
+    }
```

Comment:
```suggestion
    public boolean isScoreCalculatable() {
        return remainingNumber == 0;
    }
```

아마 Score 계산이 가능한지에 대한 여부를 나타내는 함수인 것 같은데,
Calculatable 을 네이밍에 활용해주시는 건 어떨지 제안해봅니다.

Reply:
이미 적용되어 있는거아닌가요..?

Reply:
넵 이미 잘 적용해주셨습니다.
구조를 바꿔달라기 보다는 네이밍에 관련된 사소한 피드백이었습니다.
잘 구현해주셨다보니깐 사소한것까지 언급하게 되네요.

Reply:
함수명만 바꾸는 사소한 피드백이다보니 꼭 적용해주지 않으셔도 괜찮습니다.